src/TBuffer.cpp

```diff
@@ -3136,14 +3137,19 @@ QString TBuffer::bufferToHtml(const bool showTimeStamp, const int row, const int
         return s;
     }
 
-    if ((pos < 0) || (pos >= static_cast<int>(buffer.at(row).size()))) {
+    // std:deque uses std::deque:size_type as index type which is an unsigned
+    // long int, but row (and pos) are signed ints...!
+    unsigned long cookedRow = static_cast<unsigned long>(row);
```

Comment:
How about `auto cookedRow = static_cast<unsigned long>(row)` so it doesn't repeat?

Reply:
As above...

Reply:
Done a while ago...